packages/components/src/spectrum/listView/ListView.tsx

```diff
@@ -71,8 +72,18 @@ export function ListView({
     <ListViewWrapper
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...spectrumListViewProps}
-      UNSAFE_className="dh-list-view"
+      UNSAFE_className={cl('dh-list-view', UNSAFE_className)}
+      selectedKeys={
+        selectedKeys as ListViewWrapperProps<unknown>['selectedKeys']
+      }
```

Comment:
Do we have a ticket or PR with Spectrum to include boolean in their keys? Kind of annoying we need to do this casting.

Reply:
We have a related one https://github.com/adobe/react-spectrum/issues/6103 but not one for the types.